README.adoc

```diff
@@ -395,14 +395,14 @@ Wait for your two new Pods to be in the ready state, then `curl` or visit the `{
 == Testing microservices that are running on {kube}
 
 A few tests are included for you to test the basic functionality of the microservices. If a test failure
-occurs, then you might have introduced a bug into the code. To run the tests, wait for all Pods to be
-in the ready state, then run the `mvn verify` command. The default properties defined in the `pom.xml` are:
+occurs, then you might have introduced a bug into the code. To run the tests, wait for all `Pods` to be
+in the ready state before proceeding further into this section. The default properties defined in the `pom.xml` are:
```

Comment:
Not sure if "into this section" is necessary. I believe "before proceeding further." would suffice.